README.md

```diff
@@ -1,23 +1,32 @@
-# Quarkus - Univocity Parsers
-
-## Welcome to Quarkiverse!
-
-Congratulations and thank you for creating a new Quarkus extension project in Quarkiverse!
-
-Feel free to replace this content with the proper description of your new project and necessary instructions how to use and contribute to it.
-
-You can find the basic info, Quarkiverse policies and conventions in [the Quarkiverse wiki](https://github.com/quarkiverse/quarkiverse/wiki).
-
-In case you are creating a Quarkus extension project for the first time, please follow [Building My First Extension](https://quarkus.io/guides/building-my-first-extension) guide.
-
-Other useful articles related to Quarkus extension development can be found under the [Writing Extensions](https://quarkus.io/guides/#writing-extensions) guide category on the [Quarkus.io](http://quarkus.io) website.
-
-Thanks again, good luck and have fun!
-
-## Documentation
-
-The documentation for this extension should be maintained as part of this repository and it is stored in the `docs/` directory. 
-
-The layout should follow the [Antora's Standard File and Directory Set](https://docs.antora.org/antora/2.3/standard-directories/).
-
-Once the docs are ready to be published, please open a PR including this repository in the [Quarkiverse Docs Antora playbook](https://github.com/quarkiverse/quarkiverse-docs/blob/main/antora-playbook.yml#L7). See an example [here](https://github.com/quarkiverse/quarkiverse-docs/pull/1).
\ No newline at end of file
+The Quarkus Univocity Extension adds support for
+[Univocity](https://www.univocity.com/pages/univocity_parsers_tutorial) official Univocity website.
+
+## Usage
+
+To use the extension, add the dependency to the target project:
+
+```xml
+<dependency>
+  <groupId>io.quarkiverse.jberet</groupId>
+  <artifactId>quarkus-jberet</artifactId>
+  <version>0.0.4</version>
```

Comment:
I usually use `LATEST` in my project so that users use the latest version instead of an old one. The chances of having this outdated are very high :).

Reply:
me too ! :-) but as you can see, is just a stupid cp/paste from JBeret extension